tools/ldb_cmd.cc

```diff
@@ -614,37 +715,88 @@ void LDBCommand::OverrideBaseCFOptions(ColumnFamilyOptions* cf_opts) {
     cf_opts->table_factory.reset(NewBlockBasedTableFactory(table_options));
   }
 
-  auto itr = option_map_.find(ARG_AUTO_COMPACTION);
-  if (itr != option_map_.end()) {
-    cf_opts->disable_auto_compactions = !StringToBool(itr->second);
+  cf_opts->enable_blob_files = enable_blob_files_;
+
+  int min_blob_size;
+  if (ParseIntOption(option_map_, ARG_MIN_BLOB_SIZE, min_blob_size,
+                     exec_state_)) {
+    if (min_blob_size > 0) {
```

Comment:
`min_blob_size == 0` is actually legit and frequently used (it means put all values in blob files); could you please change this check and the corresponding error message to `>= 0`?